nextflow.config

```diff
@@ -11,28 +11,77 @@ vim: syntax=groovy
  */
 
 profiles {
-    standard {
```

Comment:
There should always be a `standard` profile as it's the default. Here it can load the base config?